### ISSUE TITLE
Exception while running apm@0.0

### ISSUE THREAD

**Issue by vikasmathod** [DeborahK/Angular-GettingStarted#129]
Node version : 12.4.1
npm version 6.13.4

![image](https://user-images.githubusercontent.com/6801546/72034423-ef668280-32ba-11ea-87db-5efdfc293a6e.png)

From the log:
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start'
1 verbose cli ]
2 info using npm@6.13.4
3 info using node@v12.14.1
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle apm@0.0.0~prestart: apm@0.0.0
6 info lifecycle apm@0.0.0~start: apm@0.0.0
7 verbose lifecycle apm@0.0.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle apm@0.0.0~start: PATH: C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;D:\DeborahK\Angular-GettingStarted-master\APM-Start\node_modules\.bin;D:\books\Node js\node js;C:\Program Files\nodejs\;C:\Users\Mathod\AppData\Local\Programs\Python\Python37-32\Scripts\;C:\Users\Mathod\AppData\Local\Programs\Python\Python37-32\;C:\TclDevKit\bin;C:\Ruby200-x64\bin;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;D:\books\Node js\node js;C:\Program Files\nodejs\;C:\Program Files\Java\jdk-12\bin;C:\Users\Mathod\AppData\Roaming\npm
9 verbose lifecycle apm@0.0.0~start: CWD: D:\DeborahK\Angular-GettingStarted-master\APM-Start
10 silly lifecycle apm@0.0.0~start: Args: [ '/d /s /c', 'ng serve -o' ]
11 silly lifecycle apm@0.0.0~start: Returned: code: 1  signal: null
12 info lifecycle apm@0.0.0~start: Failed to exec start script
13 verbose stack Error: apm@0.0.0 start: `ng serve -o`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:223:5)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:223:5)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
14 verbose pkgid apm@0.0.0
15 verbose cwd D:\DeborahK\Angular-GettingStarted-master\APM-Start
16 verbose Windows_NT 6.1.7601
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
18 verbose node v12.14.1
19 verbose npm  v6.13.4
20 error code ELIFECYCLE
21 error errno 1
22 error apm@0.0.0 start: `ng serve -o`
22 error Exit status 1
23 error Failed at the apm@0.0.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

Kindly let me know what is the issue?

**Comment by nanthups** [comment]
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'dev' ]
2 info using npm@6.13.4
3 info using node@v12.14.1
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle @~predev: @
6 info lifecycle @~dev: @
7 verbose lifecycle @~dev: unsafe-perm in lifecycle true
8 verbose lifecycle @~dev: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/var/www/html/laravel-vue-spa-starter/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
9 verbose lifecycle @~dev: CWD: /var/www/html/laravel-vue-spa-starter
10 silly lifecycle @~dev: Args: [ '-c', 'npm run development' ]
11 silly lifecycle @~dev: Returned: code: 1  signal: null
12 info lifecycle @~dev: Failed to exec dev script
13 verbose stack Error: @ dev: `npm run development`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:223:5)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:223:5)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
14 verbose pkgid @
15 verbose cwd /var/www/html/laravel-vue-spa-starter
16 verbose Linux 5.3.0-28-generic
17 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "dev"
18 verbose node v12.14.1
19 verbose npm  v6.13.4
20 error code ELIFECYCLE
21 error errno 1
22 error @ dev: `npm run development`
22 error Exit status 1
23 error Failed at the @ dev script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

Same issue here

**Comment by DeborahK** [comment]
Could you provide more information on what you are doing?

This line in particular:

`10 silly lifecycle @~dev: Args: [ '-c', 'npm run development' ]`

There is no such command in the code I provided 

And from the folders it says "vue-spa-starter"?

Please provide the exact steps you are using after downloading the code from this repo.

**Comment by mvanhouten** [comment]
I was able to run and it came with some vulnerabilities, so is ok to update but make sure you run after
` npm i @angular-devkit/build-angular@0.803.24 `and it runs as it should.

Hope this helps

**Comment by DeborahK** [comment]
Closing as no additional information on the error was provided. Hopefully it was resolved by the poster. Please reopen if that is not the case. Thanks!

**Comment by Michael-Dido** [comment]
Hello,
I am having similar issue. Below is my error log:
[2020-04-20T19_19_18_257Z-debug.log](https://github.com/DeborahK/Angular-GettingStarted/files/4505690/2020-04-20T19_19_18_257Z-debug.log)

I did get the APM-Start to work on another virtual machine. So I think this is releated to some issue installing the nodeJS.  I have reinstalled nodejs but still have issue. I don't know where to look next.
